examples/dnsrewrite/Makefile

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: (c) 2025 Ring Zero Desenvolvimento de Software LTDA
```

Comment:
I don't think we need this makefile.. we only have it on the other dns filter examples for using iptables (it requires a userspace plugin)..  if we this only uses netfilter, we can avoid this file..